app/javascript/packages/password-strength/password-strength-element.spec.ts

```diff
@@ -82,6 +82,16 @@ describe('PasswordStrengthElement', () => {
     expect(input.validity.valid).to.be.false();
   });
 
+  it('updates the password aria-describedby attribute', async () => {
```

Comment:
Nice test coverage 👍  It could also be a good idea to incorporate what's expected to happen if the user clears a value after adding one (i.e. that the `aria-describedby` is removed again).